google-cloudevent-types/src/main/java/com/google/events/cloud/audit/v1/LogEntryData.java

```diff
@@ -0,0 +1,93 @@
+package com.google.events.cloud.audit.v1;
+
+import java.util.Map;
+
+/**
+ * This event is triggered when a new audit log entry is written.
+ */
+public class LogEntryData {
+    private String insertID;
+    private Map<String, Object> labels;
+    private String logName;
+    private LogEntryOperation operation;
+    private AuditLog protoPayload;
+    private String receiveTimestamp;
+    private MonitoredResource resource;
+    private String severity;
+    private String spanID;
+    private String timestamp;
+    private String trace;
+
+    /**
+     * A unique identifier for the log entry.
+     */
+    public String getInsertID() { return insertID; }
+    public void setInsertID(String value) { this.insertID = value; }
+
+    /**
+     * A set of user-defined (key, value) data that provides additional information about the
+     * log entry.
+     */
+    public Map<String, Object> getLabels() { return labels; }
+    public void setLabels(Map<String, Object> value) { this.labels = value; }
+
+    /**
+     * The resource name of the log to which this log entry belongs.
+     */
+    public String getLogName() { return logName; }
+    public void setLogName(String value) { this.logName = value; }
+
+    /**
+     * Information about an operation associated with the log entry, if applicable.
+     */
+    public LogEntryOperation getOperation() { return operation; }
+    public void setOperation(LogEntryOperation value) { this.operation = value; }
+
+    /**
+     * The log entry payload, which is always an AuditLog for Cloud Audit Log events.
+     */
+    public AuditLog getProtoPayload() { return protoPayload; }
+    public void setProtoPayload(AuditLog value) { this.protoPayload = value; }
+
+    /**
+     * The time the log entry was received by Logging.
+     */
+    public String getReceiveTimestamp() { return receiveTimestamp; }
```

Comment:
It's a pity that these are just strings. Do you anticipate us being able to improve that? (Ideally we'd return an Instant.)

Reply:
Good point.

JSONSchema does support date types, but we don't use it now and Quicktype doesn't support it.

I think we can patch the `qt` tool though to provide better types here. Filed issue:

https://github.com/googleapis/google-cloudevents/issues/87